java/com/google/flatbuffers/FlexBuffers.java

```diff
@@ -899,7 +899,7 @@ else if (cmp > 0)
      */
     public static class Vector extends Sized {
 
-        private static final Vector EMPTY_VECTOR = new Vector(ByteBuffer.allocate(0), 1, 1);
```

Comment:
Why is this necessary here, but not anywhere else?

Reply:
What do you mean by "this"?

Reply:
There's a `ByteBuffer.allocate(0)` where all others have `EMPTY_BB`

Reply:
Yes. This change replaces is with `EMPTY_BB`. 

There was no reason the have a separate allocated `ByteBuffer` there. Its just that `EMPTY_BB` was added later on during the implementation and I missed that line.

Reply:
Ah ok, didn't get that :P